thorlcr/master/thgraphmanager.cpp

```diff
@@ -1469,10 +1465,9 @@ void thorMain(ILogMsgHandler *logHandler, const char *wuid, const char *graphNam
                             {
                                 if (!streq(currentWuid, wuid))
                                 {
-                                    queryLogMsgManager()->removeJobId(thorJob.queryJobID());
-                                    LogMsgJobId thorJobId = queryLogMsgManager()->addJobId(wuid);
-                                    thorJob.setJobID(thorJobId);
-                                    setDefaultJobId(thorJobId);
+                                    activeJobName.clear();
+                                    activeJobName.set(wuid);
```

Comment:
trivial: it's fine as it is, looking at these 2 lines, it made me think 'set' should implicitly clear if it needs to.

Reply:
changed